src/ucp/proto/proto_am.inl

```diff
@@ -409,3 +409,21 @@ ucp_proto_get_short_max(const ucp_request_t *req,
             (!UCP_MEM_IS_HOST(req->send.mem_type))) ?
            -1 : msg_config->max_short;
 }
+
+static UCS_F_ALWAYS_INLINE ucp_request_t*
+ucp_proto_ssend_ack_request(ucp_worker_h worker, uintptr_t ep_ptr)
```

Comment:
`ucp_proto_ssend_ack_request`?
double `s` -> single `s`?

or does it mean `synchronized send`?

Reply:
acknowledge - sync send :)

Reply:
thanks 👍

Reply:
yes